### ISSUE TITLE
run sv3d_u success but output mp4 can't play

### ISSUE THREAD

**Issue by hengshan123** [Stability-AI/generative-models#326]
hi, I run the script success:
python scripts/sampling/simple_video_sample.py --input_path assets/000.jpg --version sv3d_u

Initialized embedder #0: FrozenOpenCLIPImagePredictionEmbedder with 683800065 params. Trainable: False
Initialized embedder #1: VideoPredictionEmbedderWithEncoder with 83653863 params. Trainable: False
Initialized embedder #2: ConcatTimestepEmbedderND with 0 params. Trainable: False
Restored from checkpoints/sv3d_u.safetensors with 0 missing and 0 unexpected keys
##############################  Sampling setting  ##############################
Sampler: EulerEDMSampler
Discretization: EDMDiscretization
Guider: TrianglePredictionGuider
Sampling with EulerEDMSampler for 51 steps:   0%|                                                                                                     | 0/51 [00:00<?, ?it/s]/home/j00668593/anaconda3/envs/py310/lib/python3.10/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
Sampling with EulerEDMSampler for 51 steps:  98%|██████████████████████████████████████████████████████████████████████████████████████████▏ | 50/51 [00:59<00:01,  1.19s/it]

And i get the output in outputs/simple_video_sample/sv3d_u. 
But the 000000.mp4 can't be played.

**Comment by wzds2015** [comment]
Same here for sv3d_p

**Comment by zijinY** [comment]
same here for sv3d_u

**Comment by hengshan123** [comment]
I check the origin result vid, this is valid. the problem maybe in imageio
![image](https://github.com/Stability-AI/generative-models/assets/7146801/33b49da4-bda8-4d8d-ba2f-6603d578c918)

**Comment by wzds2015** [comment]
confirmed the issue is from imageio. Switch to cv2 solved the problem.

**Comment by zijinY** [comment]
> confirmed the issue is from imageio. Switch to cv2 solved the problem. fix: #327

Thanks for your advise, the issue is fixed !

**Comment by hengshan123** [comment]
after install imageio-ffmpeg the problem is solved

**Comment by wyweiyangweiGh** [comment]
same question
